cole_wittrock/ruby_fun/puzzles.rb

```diff
@@ -0,0 +1,84 @@
+x = [3,5,1,2,7,9,8,13,25,32]
+
+def total arr 
+	total = 0
+	arr.each do |i|
+		total += i
+	end
+	p total
+	arr.reject! { |i| i < 10 }
+	p arr
+end
+
+# total(x)
+
+people = ["John", "KB", "Oliver", "Cory", "Matthew", "Chistopher"]
+
+def shuffle arr
+	arr.shuffle
+	arr.each do |i|
+		if i.length > 5
+		 p i
+		 end
+	end
+	arr.each do |i|
+		p i
+	end
+	# p arr.reject! {|i| i.length < 5}
+end
+# shuffle(people)
+
+alph = ["a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z"]
```

Comment:
ah, there is definitely a quicker way for this in ruby. I'd suggest using a character range: ('a'..'z')